packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -660,7 +660,8 @@ void main() {
     expect(focusNode.hasFocus, isFalse);
   });
 
-  testWidgets('use DefaultSelectionStyle for selection color', (WidgetTester tester) async {
+  testWidgets('selection color can be null with DefaultSelectionStyle as parent', (WidgetTester tester) async {
```

Comment:
Maybe rename to `EditableText does not derive selection color from DefaultSelectionStyle`